doc/source/user/tutorial-ma.rst

```diff
@@ -0,0 +1,374 @@
+    When an element of the mask is ``False``, the corresponding element of the
+    associated array is valid and is said to be unmasked. When an element of
+    the mask is ``True``, the corresponding element of the associated array is
+    said to be masked (invalid).
+    
+
+We can think of a :class:`MaskedArray <numpy.ma.MaskedArray>` as a
+combination of:
+
+- Data, as a regular :class:`numpy.ndarray` of any shape or datatype;
+- A boolean mask with the same shape as the data;
+- A ``fill_value``, a value that may be used to replace the invalid entries
+  in order to return a standard :class:`numpy.ndarray`.
+
+**When can they be useful?**
+
+There are a few situations where masked arrays can be more useful than just
+eliminating the invalid entries of an array:
+
+- When you want to preserve the values you masked for later processing, without
+  copying the array;
+- When you have to handle many arrays, each with their own mask. If the mask is
+  part of the array, you avoid bugs and the code is possibly more compact;
+- If you can't avoid or eliminate missing values, but don't want to deal with
+  :class:`NaN <numpy.nan>` (Not A Number) values in your operations.
+
+Masked arrays are also a good idea since the :mod:`numpy.ma` module also
+comes with a specific implementation of most :term:`NumPy universal functions
+(ufuncs) <ufunc>`, which means that you can still apply fast vectorized
+functions and operations on masked data. The output is then a masked array.
+We'll see some examples of how this works in practice below.
+
+**Using masked arrays to see COVID-19 data**
+
+From `Kaggle <https://www.kaggle.com/atilamadai/covid19>`_ it is possible to
+download a dataset with initial data about the COVID-19 outbreak in the
+beginning of 2020. We are going to look at a small subset of this data,
+contained in the file ``who_covid_19_sit_rep_time_series.csv``.
+
+.. code::
+   
+    >>> import numpy as np
+    >>> filename = "who_covid_19_sit_rep_time_series.csv"
+
+Let's explore the data inside this file for the first 14 days of records. To
+gather data from the ``.csv`` file, we will use the :func:`numpy.genfromtxt`
+function, making sure we select only the columns with actual numbers instead of
+the first three columns which contain location data. We also skip the first 7
+rows of this file, since they contain other data we are not interested in.
+Separately, we will extract the information about dates and location for this
+data. 
+
+.. code::
+   
+    >>> nbcases = np.genfromtxt(filename, dtype=np.int_, delimiter=",",
+    ...                         skip_header=7, usecols=range(3, 17))
+    >>> dates = np.genfromtxt(filename, dtype=np.unicode_, delimiter=",",
+    ...                       max_rows=1, usecols=range(3, 17))
+    >>> locations = np.genfromtxt(filename, dtype=np.unicode_, delimiter=",",
+    ...                           skip_header=7, usecols=(0, 1))
```

Comment:
If possible,
```suggestion
    >>> nbcases = np.genfromtxt(filename, dtype=np.int_, delimiter=",",
    ...                         skip_header=7, usecols=range(3, 17), encoding='utf8')
    >>> dates = np.genfromtxt(filename, dtype=np.unicode_, delimiter=",",
    ...                       max_rows=1, usecols=range(3, 17), encoding='utf8')
    >>> locations = np.genfromtxt(filename, dtype=np.unicode_, delimiter=",",
    ...                           skip_header=7, usecols=(0, 1), encoding='utf8')
```
Good practice when loading international data to use character encodings which can hold international names

Reply:
Sure, same as above, it worked here but maybe it's because my whole system is UTF-8 since that's the standard in Brazil